general/cisetup/utils.go

```diff
@@ -33,27 +32,6 @@ const (
 	MinSupportedXrayVersion        = "3.20.0"
```

Comment:
I think we need to delete those too. (MinSupportedArtifactoryVersion, MinSupportedXrayVersion)